br/pkg/lightning/backend/local/local.go

```diff
@@ -424,11 +463,10 @@ func (local *local) Close() {
 		engine.Close()
 		engine.unlock()
 	}
-	local.conns.Close()
-	local.bufferPool.Destroy()
```

Comment:
Why remove this

Reply:
This was an accident during merge master.🤣